python/lib/repository_standards.py

```diff
@@ -0,0 +1,201 @@
+"""
```

Comment:
Not a massive fan of the string docs for the classes and functions 📝 

I believe they tend to get filtered out and forgotten by people when they're placed on most things 👀 

Different if it's placed on 1/100 functions, people will tend to notice the exception and spend the extra time to read it

Reply:
I can see their place if this was ever imported or used by anyone else, but maybe not relevant here. I was following the PEP 8 style guide for public functions and classes - I could remove the module string, though.

Reply:
I've noticed a couple of docstrings on private methods here, but they feel relevant as there is some instruction/purpose.

Reply:
I agree with both side heres, as Connor said people may skip it or leave it to and the content becomes out of date, on the other hand it adds the documentation into the file rather than in a seperate doc/wiki/site. In a production environment in the past I've had to use doxygen to generate the code docs that were passed onto the customer as part of the deliverable. As were not bound by any regulations to have them it comes down to personal choice to include them or not

Reply:
Agree that they add value for packages intended to be used by the general public ✅ (even when the docs seem rudimentary) 

But when the intended audience is us (dare I say, the domain experts 😂) - they usually quickly degrade and admittedly, I will tend to mentally blur them out more often then notice them